data/json/npcs/missiondef.json

```diff
@@ -461,6 +461,31 @@
       "failure": "Quitting already?"
     }
   },
+  {
+    "id": "MISSION_KILL_NEMESIS",
+    "type": "mission_definition",
+    "name": { "str": "Kill doppelganger" },
+    "goal": "MGOAL_KILL_MONSTER_SPEC",
+    "monster_species": "NEMESIS",
+    "monster_kill_goal": 1,
+    "difficulty": 5,
+    "value": 200000,
+    "urgent": true,
+    "start": "kill_nemesis",
+    "end": { "effect": [ { "u_lose_trait": "HAS_NEMESIS" } ] },
+    "origins": [ "ORIGIN_GAME_START" ],
+    "dialogue": {
+      "describe": "The eater of the dead… something was ripping zombies to shreds and only leaving a few scattered limbs…",
+      "offer": "A few days ago another survivor and I were trying to avoid the cities by staying in the woods during the day and foraging for gear at night.  It worked well against the normal zed's but one night something caught onto our trail and chased us for ten minutes or so until we decided to split up and meet-up back here.  My buddy never showed up and I don't have the means to kill whatever it was.  Can you lend a hand?",
```

Comment:
Did you copy-paste the jabberwock lines?

Reply:
I did. Documentation left me under the impression these descriptions are requires, though they will never appear in game because this mission is not assigned by an NPC. 

I suppose I should modify them to "n/a" or something similar

Reply:
I don't think you need them, the missions you get from starting scenarios don't have them.

Reply:
Looks like you are correct and I misread the documentation! Removing all the dialog options.